src/db/models/verifyDevice.js

```diff
@@ -0,0 +1,26 @@
+import firebase from 'firebase-admin';
+import serviceAccount from './fireboy-79d85-firebase-adminsdk-eq2r2-07f67da5cb.json';
+
+class verifyDevice {
+  constructor(model) {
+    this.Model = model;
+    this.fb = firebase.initializeApp({
```

Comment:
The firebase connection creds used here are static, there is not way to switch between a test, staging, or production firebase db.

I expect the creds here to be configurable via the environment,

Reply:
i'll work on a better way